### ISSUE TITLE
Statistics/Information:

### ISSUE THREAD

**Issue by Hydro5** [mpv-player/mpv#1760]
Information about the media file being played should be accessible with a key by default. 

This can be set up now, but I think this feature should be default. 

http://imgur.com/SqHBJov

http://paste.fedoraproject.org/206783/28026367/

**Comment by haasn** [comment]
In its current form, this is too messy to consider inclusion. We discussed ways to approach this on IRC, what I came up with is something that could look like this:

1. Add a command to render out a “property list” or associative array, in the sense that it would internally format the indentation levels and bold the names etc.

2. Either expose a default property collection for this, or make it accessible from a Lua API and provide a lua script to generate a nice property list like this one or something.

Basically, the \\tag spam needs to go, and the idea here would be to move it to the logic for 1. so it wouldn't have to be repeated throughout the list of what you *actually* want to display.

Furthermore, the lua script API would be reusable for all kinds of debugging.
